mobile/response_test.go

```diff
@@ -25,3 +25,15 @@ func TestPrepareJSONResponseErrorWithError(t *testing.T) {
 	data := prepareJSONResponse("0x123", errors.New("some error"))
 	require.Contains(t, data, `{"error":{"message":"some error"}}`)
 }
+
+func TestCompressAnyKey(t *testing.T) {
+	key := CompressPublicKey("0x045596a7ff87da36860a84b0908191ce60a504afc94aac93c1abd774f182967ce694f1bf2d8773cd59f4dd0863e951f9b7f7351c5516291a0fceb73f8c392a0e88")
```

Comment:
Hi @qfrank : Yes sounds good!

Reply:
@qfrank : `TestComressAnyKey` was actually not needed and put there so that I could see whats the output..

Reply:
emm.. i still think it's needed as it will also let ppl more easier to know how to transfer from desktop key into mobile key, does this make sense? @siddarthkay

Reply:
> emm.. i still think it's needed as it will also let ppl more easier to know how to transfer from desktop key into mobile key, does this make sense? @siddarthkay

Oh for that I have added `TestDeserializeAndCompressKeyApi`